src/pages/Browse/index.tsx

```diff
@@ -191,7 +191,7 @@ export class BrowseBase extends React.Component<Props> {
     if (!file || !version) {
       return <Loading message={gettext('Loading content...')} />;
     }
-    if (file.type === 'image' && file.downloadURL) {
+    if (file.fileType === 'image' && file.downloadURL) {
```

Comment:
That seems redundant, no?

Reply:
I suggest we revert that.

Reply:
I did it because I agreed with @eviljeff's observation that `type` was a bit ambiguous. Perhaps that's not even what he was saying, but we do have a few different `type` fields on different objects, and I like this change. I know it seems a bit redundant, but `type` for file could imply `mimeType`, or type based on extension, or maybe even something else that a reader might infer. To me `fileType` says "what is the type of this file?"

If you feel really strongly about it I can revert it, but I actually like it and don't think it's worth the time and effort to revert it.